metals/src/main/scala/scala/meta/internal/metals/debug/DebugProvider.scala

```diff
@@ -427,9 +427,16 @@ class DebugProvider(
           if params.getDataKind == b.DebugSessionParamsDataKind.SCALA_MAIN_CLASS =>
         json.as[b.ScalaMainClass] match {
           case Success(main) if params.getTargets().size > 0 =>
+            val javaBinary = buildTargets
+              .scalaTarget(params.getTargets().get(0))
+              .flatMap {
+                _.jvmHome.map(path => path.toAbsolutePath.resolve("bin/java"))
```

Comment:
Maybe use `JavaBinary.path` or even `allPossibleJavaBinaries`? Otherwise you might still want to include `java.exe` .

Reply:
Good idea!